package.json

```diff
@@ -20,7 +20,8 @@
     "node": ">= 0.8"
   },
   "scripts": {
-    "test": "jshint lib index.js && jscs lib index.js && mocha -t 5000 -b -R spec test/index"
+    "test": "jshint lib index.js && jscs lib index.js && mocha -t 5000 -b -R spec test/index",
+    "cover": "nyc --reporter=lcov --reporter=text-summary npm test"
```

Comment:
Can you get this to print the coverage in the CI?  I'm lazy and don't like fetching changesets to run locally.